baram/iam_manager.py

```diff
@@ -1,4 +1,6 @@
 import boto3
+import fire
+import pprint
 
 
 class IAMManager(object):
```

Comment:
No Test Cases?

Reply:
thank you for pointing out. will pytest cases  in next commit